community.md

```diff
@@ -2,54 +2,61 @@
 
 [!INCLUDE[](~/includes/blazor-preview-notice.md)]
 
-Below is a list of projects and resources from the Blazor community (links in each section are alphabetical). If you know about another Blazor related project that isn't listed here, please let us know and we'd be happy to add it to the list.
+Below is a list of projects and resources from the Blazor community (links in each section are alphabetical). If you know about a Blazor project that isn't listed, please let us know. We're happy to add it to the list.
 
 ## State management
```

Comment:
Can we move the State management stuff into the libraries section? I don't think it really needs to be a separate section